external-crates/move/crates/move-analyzer/trace-debug/src/extension.ts

```diff
@@ -492,7 +499,7 @@ async function getTracedFunctionInfo(traceFilePath: string): Promise<TracedFunct
     if (!traceContent) {
         throw new Error(`Cannot decompress trace file '${traceFilePath}'`);
     }
-    const traceLines = traceFilePath.split('\n');
```

Comment:
Really unfortunate result of a previous change utilizing variable renaming... Obviously we should be operating on trace content here and not on its path... If only TS allowed these types of things to be caught by the compiler...